src/templates/ensembles.js

```diff
@@ -17,7 +16,7 @@ export const EnsemblesTemplate = ({
   header: { topText, bottomText },
   subheading,
   about,
-  button: { buttonText, buttonLink },
+  concertAttire: { attireHeading, attireContent, attireDrawer  },
```

Comment:
If we de-structure the elements here then if they don't have them the admin panel throws an error. I suggest adding in some default props and / or the default property on config.yaml